run.sh

```diff
@@ -6,14 +6,16 @@ set -e
 
 readonly FILE_SCRIPT="$(basename "$0")"
 readonly DIR_SCRIPT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-readonly GIT_COMMIT="$(git --git-dir=${DIR_SCRIPT}/.git log -1 --format=%H | cut -c1-8)"
+pushd "${DIR_SCRIPT}" &> /dev/null
+readonly GIT_COMMIT="$(git log -1 --format=%H | cut -c1-8)"
 readonly VARISCITE_REGISTRY="ghcr.io/varigit/var-host-docker-containers/yocto-env"
 
-cd ${DIR_SCRIPT}
+cd "${DIR_SCRIPT}"
```

Comment:
This is no longer needed if the script will `cd "${DIR_SCRIPT}"` before `readonly GIT_COMMIT=...`